server/tests/interpreter_test.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+var Interpreter = require('../interpreter');
+var autoexec = require('../autoexec');
+var testcases = require('./testcases');
+
+for (var i = 0; i < testcases.length; i++) {
+  var tc = testcases[i];
+
+  if (!('expected' in tc)) {
+    console.log('SKIP\t%s', tc.name);
+    continue;
+  }
+  
+  var interpreter = new Interpreter()
+  interpreter.appendCode(autoexec);
+  interpreter.run();
+
+  var err = undefined;
```

Comment:
Yes there is, because 'var' is not block scoped, and needs to be undefined at this point in the loop.

Changed to let, along with other styleguide improvements (this file only)